### DISCUSSION TITLE
Explorer.exe crashes when I run this

### DISCUSSION THREAD

**Discussion by erspark2002** [valinet/ExplorerPatcher#735] [General]
When launching ep_setup.exe as admin (or without), the taskbar disappears (a sign of explorer.exe crashing), then reloads again, and the setup program never runs.

ep_setup.exe version: 22000.376.40.21

OS Name	Microsoft Windows 11 Pro
Version	10.0.22000 Build 22000

**Comment by valinet** [comment]
Please read [this](https://github.com/valinet/ExplorerPatcher/wiki/Reporting-problems) and try the steps mentioned there. In particular, you can enable full user mode crash dumps in order to attempt to determine why  it happens, if it does. Also, try to manually run explorer.exe. And also, try with the latest pre-release or the one before it. Thanks.
